client/OidcClient.js

```diff
@@ -0,0 +1,99 @@
+import { EventEmitter } from 'events';
+import path from 'path';
+
+const isProduction = process.env.NODE_ENV === 'production';
+const SILENT_REFRESH_TIMEOUT = 5000;
+
+class OidcClient {
+  constructor() {
+    this.enabled = process.env.OIDC_ENABLED;
+    this.baseUrl = path.join(process.env.API_PATH, 'oidc');
```

Comment:
Can this be done without importing the node.js 'path' module on the client side?

Reply:
We already have `path` used for `extname` in `tce-video` so it's not anything new in the bundle and it's safer to build paths with `path#join`. Do you have an idea of how to do it without it on the client side? 🤔

Reply:
I think we used `url-join` in these cases. I can not for the life of me remember the entire discussion and with whom I had it but it was about avoiding using Node.js modules on the client-side. But, if it is already used on the client-side then keep it.

Reply:
You're right, better than not to reinforce the incorrect convention by introducing another precedent so I'll change it to `url-join`.